tools/generate_schema_interface.py

```diff
@@ -187,27 +183,67 @@ def print_(*args, **kwargs):
             if prop.type not in templates:
                 raise ValueError("No template for type={0}".format(prop.type))
             outfile = os.path.join(path, '{0}.py'.format(key.lower()))
-            print_(" - Writing {0}".format(os.path.basename(outfile)))
+            print(" - Writing {0}".format(os.path.basename(outfile)))
             with open(outfile, 'w') as f:
                 f.write(templates[prop.type].render(cls=prop))
 
-        print_("Writing tests to {0}".format(testpath))
+    def write_interface_tests(self, path=None):
+        # Make sure the path is valid
+        if path is None:
+            path = getpath('..', 'altair', 'schema', '_interface', 'tests')
+        if not os.path.exists(path):
+            os.makedirs(path)
+
+        print("Writing tests to {0}".format(path))
 
         # Write test Init File
         template = self.templates.get_template('__init__.py.tpl')
         header = 'Auto-generated tests for Vega-Lite Schema wrappers'
-        print_(" - Writing __init__.py")
-        with open(os.path.join(testpath, '__init__.py'), 'w') as f:
+        print(" - Writing __init__.py")
+        with open(os.path.join(path, '__init__.py'), 'w') as f:
             f.write(template.render(objects=[], header=header))
 
         # Write test file
         template = self.templates.get_template('test_instantiations.py.tpl')
         classes = [prop for key, prop in sorted(self.definitions.items())]
-        print_(" - Writing test_instantiations.py")
-        with open(os.path.join(testpath, 'test_instantiations.py'), 'w') as f:
+        print(" - Writing test_instantiations.py")
+        with open(os.path.join(path, 'test_instantiations.py'), 'w') as f:
             f.write(template.render(classes=classes))
 
+    def write_wrappers(self, path=None):
+        # make sure the path is valid
+        if path is None:
+            path = getpath('..', 'altair', 'schema', '_wrappers')
+        if not os.path.exists(path):
+            os.makedirs(path)
+
+        print("Writing wrappers to {0}".format(path))
+
+        classes = [('PositionChannel', 'PositionChannelDef'),
```

Comment:
Let's create a `CLASS_ALIASES` constant at the top of the file and also include the other aliases that are in the api.